content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md

```diff
@@ -134,6 +134,7 @@ As a node operator, you should upgrade to the [latest released version of Corda]
 In this release:
 
 * Corda Shell has been removed to its own repository for improved security. You can now use a standalone shell outside of the node, or from within the node's drivers. You can read more about using the standalone shell [here](./node/operating/shell.html#the-standalone-shell). For information on adding the shell to the node's drivers, see [Upgrading a node to Corda Enterprise Edition 4.9](node-upgrade-notes.html).
```

Comment:
```suggestion
* Corda shell has been removed to its own repository for improved security. You can now use a standalone shell outside of the node, or from within the node's drivers. For more information about using the standalone shell, see [The standalone shell](./node/operating/shell.html#the-standalone-shell). For information on adding the shell to the node's drivers, see [Upgrading a node to Corda Enterprise Edition 4.9](node-upgrade-notes.html).
```
Do not use words such as "here", "click here", and so on as links. Use more meaningful links.